k8s/models/common.py

```diff
@@ -50,6 +50,24 @@ class ObjectMeta(Model):
     selfLink = ReadOnlyField(six.text_type)
     uid = ReadOnlyField(six.text_type)
 
+class ObjectReference(Model):
+    apiVersion = Field(six.text_type)
+    fieldPath = Field(six.text_type)
+    kind = Field(six.text_type)
+    name = Field(six.text_type)
+    namespace = Field(six.text_type)
+    resourceVersion = Field(six.text_type)
+    uid = Field(six.text_type)
+
+class ListMeta(Model):
```

Comment:
I don't think we need this, since we normally don't define the various `List` objects (see comment on `CronJobList`).